### ISSUE TITLE
iOS Accessibility: cannot tap on sidebar when using voiceover

### ISSUE THREAD

**Issue by jfayre** [home-assistant/home-assistant-iOS#456]
I am blind and use the Voiceover accessibility feature on my iPhone.
When attempting to use the UI from within the Home Assistant app, double-tapping on the "sidebar toggle" button doesn't bring up the sidebar. I need to use the Voiceover double-tap and hold gesture to pass the gesture through to the application. This shouldn't be necessary.
Once I am able to get the sidebar to come up, I need to use the same procedure to activate any of the items in the sidebar.
I am running HASS 0.100.1 and the latest version of the app from testflight.

**Comment by TomBrien** [comment]
Hi, does the same problem exist if you access Home Assistant directly through Safari?

**Comment by jfayre** [comment]
I just checked and this does happen with Safari as well. Should I open this bug on the main HA github instead?

**Comment by TomBrien** [comment]
I'll try to move this over to the frontend repo now. Cheers
